mod-orders-storage/examples/receiving_history_collection.sample

```diff
@@ -0,0 +1,21 @@
+{
+  "receiving_history":
+    [
+      {
+        "id": "5e317dc2-deeb-4429-b2a1-91e5cd0fd5f7",
+        "caption": "Tutorial Volume 10",
+        "comment": "Special Edition",
+        "item_id": "1c92433e-916d-452e-ac76-5268d0ec3f66",
+        "po_line_id": "c1498090-538a-4470-9525-27e4e0c74b07",
+        "po_line_number": "268758-03",
+        "received_date": "2018-10-09T00:00:00.000Z",
+        "receiving_note": "ASD FGH ABCDEFGHIJKLMNOPQ",
+        "receiving_status": "Expected",
+        "supplement": true,
+        "title": "Kayak Fishing in the Northern Gulf Coast"
+      }
+    ],
+  "total_records": 1,
+  "first": 1,
```

Comment:
shouldn't first/last be 0 here?  or is do these fields really use 1-based indexing?

Reply:
`total_records` `first` `last` start from 1. It is consistent with all other schemas we have till now.

Reply:
That seems confusing to me, but if that's really what the rest of FOLIO is doing I guess we should be consistent

Reply:
Now that I look I can't find any other places in FOLIO (besides acquisitions) that use this first/last.  This means, whatever precedent there is, it's our own, meaning we could change it if we wanted to.  The caveat there is that if we're going to do so we should soon, before other teams start integrating with our APIs.

@komick @bonthebeerman:  Do either of you know the history here?  Is this a carry-over from the JD days?  Is there a reason it's 1-based?

Reply:
Let's not let this hold us up... I'll create a user story to track this and hopefully we can find an answer and get some clarity on this.

Reply:
FYI:  https://issues.folio.org/browse/MODORDSTOR-37